src/SprykerEco/Zed/Payolution/Communication/Plugin/Oms/Command/AbstractPayolutionCommandPlugin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payolution\Communication\Plugin\Oms\Command;
+
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+abstract class AbstractPayolutionCommandPlugin extends AbstractPlugin
```

Comment:
Abstract classes are not extendable, please try to avoid them in core development, especially in plugin context.